libcontainer/intelrdt/intelrdt.go

```diff
@@ -166,22 +205,46 @@ type Manager interface {
 
 // This implements interface Manager
 type intelRdtManager struct {
-	mu     sync.Mutex
-	config *configs.Config
-	id     string
-	path   string
+	mu        sync.Mutex
+	config    *configs.Config
+	id        string
+	path      string
+	groupType string
```

Comment:
Should we have a separate type defined?

And, actually when thinking more about this: should this merely be something like
```
	ctrlGroupIsExclusiveToThisContainer bool
```

(I know that name^ is really bad but you get the intention). Then, if this is false create/read container-specific sub-MON groups. If this is true, read stats directly from the `mon_data` dir of the CTRL group.